scripts/bootstrap/Makefile.bootstrap

```diff
@@ -130,9 +130,9 @@ $(RELEASE_ROOT)/_build/default/esy-build-package/bin/esyBuildPackageCommand.exe:
 $(RELEASE_ROOT)/bin/esy-install.js:
 	@$(MAKE) -C esy-install BUILD=../$(@) build
 
-$(RELEASE_ROOT)/bin/fastreplacestring-darwin:
+$(RELEASE_ROOT)/bin/fastreplacestring:
 	@mkdir -p $(@D)
-	@cp $$(esy which fastreplacestring) $(@)
+	@cp $(PWD)/bin/fastreplacestring $(@)
```

Comment:
Do you need this line if you have `copy-fastreplacestring.js` below?

Reply:
At least for now - the `copy-fastreplacestring.js` copies from `esy/node_modules/fastreplacestring/.bin/fastreplacestring-win{arch}.exe` to `esy/bin/fastreplacestring`. Then, the `makefile` copies `esy/bin/fastreplacestring` to `_release/bin/fastreplacestring`.

It's convoluted but my plan is once we have an npm package that has `esy` working on Windows, we can snap back to using the same makefile across platforms.